pulp_rpm/app/models.py

```diff
@@ -721,3 +721,23 @@ class Modulemd(Content):
     dependencies = models.TextField(default='[]')
     artifacts = models.TextField(default='[]')
     packages = models.ManyToManyField(Package)
+
+
+class ModulemdDefaults:
```

Comment:
It's a content type, so it should be inherited from the `Content` class provided by pulpcore